src/slider.toit

```diff
@@ -26,9 +28,40 @@ class Slider extends CustomElement:
   thumb_max_/int?
   boundary_/int := 0
 
-  type -> string: return "vertical-slider"
+  type -> string: return "slider"
 
-  constructor --x/int?=null --y/int?=null --w/int?=null --h/int?=null --background-hi=null --background-lo=null --value/num?=null --min/num?=0 --max/num?=100 --thumb_min/int=0 --thumb_max/int?=null --horizontal/bool=false:
+  /**
+  Constructs a new slider element.
+  You can provide a background to draw when the slider is above a certain level,
```

Comment:
This paragraph is relevant for the `background_hi` and `background_lo` parameters. They should be referenced in here.